---
layout: post
title: "16S v1-3 sequencing on MiSeq"
description: ""
category: ""
tags: []
---
{% include JB/setup %}

> This is our protocol for sequencing 16S amplicons on the MiSeq.  We use the Nextera XT barcodes as this is a quick and simple way of multiplexing 96 samples on each run.  We have chosen v1-3 as it was a popular on the 454 but has only recently become suitable for the MiSeq with the introduction of the v3 reagent kits.

##Primers

For our initial experiments targeting v1-3 we chose the popular primer combination 27F/519RF. Our primers also included ambigous bases to improve their coverage. This is a two-step PCR method, which has advantages in terms of scalability. In the first step you amplify your target region whilst adding Nextera XT overhang adapters. In the second step you add the barcoded adapters and perform additional amplification. The great thing about this method is you only need to order two primers to get started with 96 samples and you have the flexibilty to add further genes/targets down the road. Using the Nextera XT barcodes means you are less likely to have problems with poor % passing filter or % identified reads which can happen with homebrew constructs.


| Forward primer 5'-3'              |                      |                                                         |
|-----------------------------------|----------------------|---------------------------------------------------------|
| Illumina overhang adapter         | 27F                  | Forward primer                                          |
| TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG | AGAGTTTGATYMTGGCTCAG | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG**AGAGTTTGATYMTGGCTCAG** |


| Reverse primer 5'-3'               |                    |                                                        |
|------------------------------------|--------------------|--------------------------------------------------------|
| Illumina overhang adapter          | 519R               | Reverse primer                                         |
| GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG | GWATTACCGCGGCKGCTG | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG**GWATTACCGCGGCKGCTG** |

Our oligos are supplied by [Alta Bioscience](http://www.altabioscience.com/), Birmingham. Primers are usually lyophilised (freeze-dried) before shipping, so they need resuspending in PCR grade water (Roche). All necessary PCR equipment (e.g. pipettes, plastic ware) should be prewashed with 70% ethanol and left under UV light for 90min in a laminar flow hood. PCR reagents should also be mixed in a laminar flow hood.

|Reagent             | Volume per reaction|
|:-------------------|:-------------------|
|10x reaction buffer | 2.5                |
|MgCl2 50mM          | 0.65               |
|dNTPs 10mM          | 0.5                |
|F primer 10uM       | 0.5                |
|R primer 10uM       | 0.5                |
|Polymerase 5u/ul    | 0.25               |
|PCR-grade water     | 18.1               | 
|DNA                 | 2                  |
|Final volume        | 25                 |


|Step            | Temp (oC) | Time (s) | Repeats|
|:---------------|:----------|:---------|:-------|
|Pre-heat        | 98        | 120      |        |
|Denaturation    | 98        | 30       | 20     |
|Annealing       | 55        | 30       | 20     |
|Extension       | 72        | 90       | 20     |
|Final extension | 72        | 300      |        |
|Hold            | 4         |          |        |


First step PCR required to use smaller amount of MgCl2 to make reaction less specific and give primers chance to bind to those gDNA 16S regions which are not fully homologus to primer sequence. It is important to balance amount of MgCl2 as to small amount can cause amplification of random secondary products from bacterial DNA. Moreover, when analysing metagenomic clinical samples low concentration of MgCl2 can amplify human genes. Concentrations of remaining reagents were followed as recommended by the manufacturer of the polymerase.
Each reaction was carried in triplicate to decrease amplification bias.

Temperature of PCR was followed as recommended by the manufacturer of polymerase with regard of annealing temp of primers.
First and second step PCR was performed with the same PCR conditions, with exception of amount of repeats respectively 20 and 10 repeats.

![First step PCR](/images/130911_step_one_pcr.png)

Picture of first step PCR amplicons before AmPure SPRI bead clean up. 

Before first step PCR amplicons were PCR'ed again with Illumina barcodes we cleaned them with use of magnetic beads to remove: primer dimers, secondary products, remainings of gDNA etc. Furthermore, use of specific ratio (0.6x) of magnetic beads allowed to keep only these amplicons which we have been interested in (500-600bp).

Second step PCR required to use higher amount of MgCl2 to make sure that Illumina primers will bind to the overhang adapters and amplify the whole V1-V3 region.  Smaller amount of MgCl2 caused primers to bind in non-specific region of ~500bp, 16S amplicon region and formed  ~200bp secondary product which were too difficult to remove with AmPure SPRI beads. Again reaction for each sample was carried in triplicate to decrease amplification bias and subsequently increase sequencing quality.

Temperature of reaction was followed as in the first step PCR, the annealing temperature for Illumina primers was followed as recommended in Nextera XT protocol.
Gel electrophoresis was performed on second step PCR 

![Second step PCR](/images/130911_step_two_pcr.png)

SPRI beads clean up was performed again with  the same ratio as previously described.

Kappa Syber Green quantification:
qPCR reaction was set up in sterile laminar flow hood using SYBR Green Kappa Kit and processed with use of Agilent Mx3005P machine.
Reaction volume, temperatures and sample dilution were exactly followed as recommended by the manufacturer with additional step of dissociation curve.
Each reaction was set up in triplicate and treated collectively by the MxPro software

![qPCR results](/images/130911_KAPA_qpcr.png)

Fig. 1 Indicates Ct and exact quantity values for each sample and standards

![Amplification plots](/images/130911_KAPA_amp_plot.png)

Fig. 2 SYBR Green qPCR amplification plot represents Ct values for samples and standards in graphical way. (Its just another way of looking at the results)


