---
layout: post
title: "16S v1-3 sequencing on MiSeq"
description: ""
category: "metagenomics"
tags: [16S, MiSeq, Nextera XT, PCR, Amplicons]
---
{% include JB/setup %}

> This is our protocol for sequencing 16S amplicons on the MiSeq.  We use the Nextera XT barcodes as this is a quick and simple way of multiplexing 96 samples on each run.  We have chosen v1-3 as it was a popular on the 454 but has only recently become suitable for the MiSeq with the introduction of the v3 reagent kits.

##Primers

For our initial experiments targeting v1-3 we chose the popular primer combination 27F/519RF. Our primers also included ambigous bases to improve their coverage. This is a two-step PCR method, which has advantages in terms of scalability. In the first step you amplify your target region whilst adding Nextera XT overhang adapters. In the second step you add the barcoded adapters and perform additional amplification. The great thing about this method is you only need to order two primers to get started with 96 samples and you have the flexibilty to add further genes/targets down the road. Using the Nextera XT barcodes means you are less likely to have problems with poor % passing filter or % identified reads which can happen with homebrew constructs.


| Forward primer 5'-3'              |                      |                                                         |
|-----------------------------------|----------------------|---------------------------------------------------------|
| Illumina overhang adapter         | 27F                  | Forward primer                                          |
| TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG | AGAGTTTGATYMTGGCTCAG | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG**AGAGTTTGATYMTGGCTCAG** |


| Reverse primer 5'-3'               |                    |                                                        |
|------------------------------------|--------------------|--------------------------------------------------------|
| Illumina overhang adapter          | 519R               | Reverse primer                                         |
| GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG | GWATTACCGCGGCKGCTG | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG**GWATTACCGCGGCKGCTG** |

Our oligos are supplied by [Alta Bioscience](http://www.altabioscience.com/), Birmingham. Primers are usually lyophilised (freeze-dried) before shipping, so they need resuspending in PCR grade water. All necessary PCR equipment (e.g. pipettes, plastic ware) should be prewashed with 70% ethanol and left under UV for 90 minutes in a laminar flow hood. PCR reagents should also be mixed in a laminar flow hood.

##First step PCR

|Reagent             | Volume per reaction|
|:-------------------|:-------------------|
|10x reaction buffer | 2.5                |
|MgCl2 50mM          | 0.65               |
|dNTPs 10mM          | 0.5                |
|F primer 10uM       | 0.5                |
|R primer 10uM       | 0.5                |
|Polymerase 5u/ul    | 0.25               |
|PCR-grade water     | 18.1               | 
|DNA                 | 2                  |
|Final volume        | 25                 |


|Step            | Temp (oC) | Time (s) | Repeats|
|:---------------|:----------|:---------|:-------|
|Pre-heat        | 98        | 120      |        |
|Denaturation    | 98        | 30       | 20     |
|Annealing       | 55        | 30       | 20     |
|Extension       | 72        | 90       | 20     |
|Final extension | 72        | 300      |        |
|Hold            | 4         |          |        |

It is important to optimise the PCR conditions to suit your sample. Our samples were heavily contaminated with human DNA so in order to prevent excessive amplification of off-target human products, annealing temperature and MgCl2 concentration were optimised. This was done using a gradient PCR with a titration of MgCl2. A good starting point is an annealing temperature 5oC lower than the Tm of the primers and 1.5-4ul of MgCl2. We used BIOTAQ polymerase from Bioline with the recommended concentration of primers and dNTPs. All reactions were performed in triplicate and pooled to increase library diversity.

![First step PCR](/images/130911_step_one_pcr.png)

DNA gel showing the first step PCR amplicons before SPRI clean-up. 

PCR products then need to be cleaned up using SPRI beads. The v1-3 amplicon is around 550bp in length so we used a ratio of 0.6x SPRI to clean-up the sample. This removes primers, polymerase and products below 400bp.


##Second step PCR

As with the first-step the reaction conditions need to be optimised, however given that the overhang adapters should have an exact match for the barcoded adapters the optimum annealing temperature and MgCl2 concentration are likely to be higher. The Illumina barcode adapter plate is very useful at this step, as it holds the barcode tubes in position allowing you to pipette them with a multi-channel pipette. After a further 10 cycles on PCR the products (now 650bp) were run on a gel.

![Second step PCR](/images/130911_step_two_pcr.png)

Products were cleaned-up again using 0.6x SPRI. A pool was made with equal volumes of each sample then quantified using SYBR Green qPCR with a KAPA library quantification kit.

##KAPA quantification

The 96-well plate was made up in a laminar flow hood according to the manufacturers protocol and run on the Agilent Mx3005P instrument. Each reaction was set up in triplicate and treated collectively by the MxPro software.

![qPCR results](/images/130911_KAPA_qpcr.png)

qPCR results displaying Ct values and initial quantity for samples and standards.

![Amplification plots](/images/130911_KAPA_amp_plot.png)

Amplification plots showing fluorescence by cycle for samples and standards.

##MiSeq sequencing

Although recent improvements to RTA have overcome many of the problems assocaiated with sequencing low diversity libaries such as this, it is still necessary to use a spike-in.  As the amplicon pool uses dual indexes it's a good idea for your spike in to be dual indexed.  We used a Nextera XT library taken from a previous plate, choosing an unused index combination. Using the quantification results, dilute both the amplicon pool and the spike in to 2nM with EB. Then add 8ul of the amplicon pool and 2ul of spike in (for 20% spike in) to an Eppendorf tube containing 10ul 0.2N freshly made NaOH, mixing up and down with a pipette. Leave this to incubate at room temperature for 5 minutes before adding 980ul of pre-chilled HT1. This gives you a 20pM denatured library, in order to target 15M reads dilute this to 14pM with HT1 and load into a thawed MiSeq v3 reagent cartridge. 




